distributed/comm/registry.py

```diff
@@ -60,10 +60,32 @@ def get_local_address_for(self, loc):
 def get_backend(scheme):
     """
     Get the Backend instance for the given *scheme*.
+
+    New backends can be registered using setuptools entry_points, e.g.:
```

Comment:
It seems that `get_backend` isn't included in `docs/source/communications.rst`. Could you add it and a `.. autofunction` for it?

Reply:
I moved the documentation to `communications.rst`